clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -114,6 +117,42 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(config_path):
+            """ if a configuration file exists,
+            parse it and accordingly deploy models and applications
+
+            Parameters
+            ----------
+            config_path : str
+                Path to the configuration file.
+            """
+            if(config_path):
+                with open(config_path, "r") as stream:
+                    config = yaml.load(stream)
+                    for name in config:
+                        if(name == "links"):
+                            for link in config[name]:
+                                self.link_model_to_app(config[name][link]["app_name"], config[name][link]["model_name"])
+                        else:
+                            obj_type = config[name]["obj_type"]
```

Comment:
We should check that required keys are present and throw `ClipperException` errors if they are absent (https://github.com/ucbrise/clipper/blob/137cc852a5c9ed5cca11c6c2f89f1e708a561198/clipper_admin/clipper_admin/exceptions.py#L1).

Using python's inspect module (https://docs.python.org/3/library/inspect.html), you can determine the set of parameters associated with the `deploy_model`, `register_application`, and `link_model_to_app` methods, as well as which parameters have default values (see https://stackoverflow.com/questions/218616/getting-method-parameter-names-in-python).